nequip/utils/aot.py

```diff
@@ -60,7 +60,7 @@ def aot_export_model(
     assert out_path == output_path
 
     # === sanity check ===
-    if not skip_model_check:
+    if os.environ.get("NEQUIP_SKIP_AOTI_MODEL_CHECK", "0") == "0":
```

Comment:
just convincing myself:  if the env var isn't defined, this `.get` returns `"0"`, and so the test _does_ get run by default, correct?  Want to be very sure we aren't accidentally turning off the test

Reply:
Yep! I can change it to `!= "1"` if preferred

Reply:
Well, I guess `!= "1"` is a little more explicit

Reply:
Updated!